lib/primitives/mtx.js

```diff
@@ -1911,6 +1911,98 @@ class CoinSelector {
     this.fund();
   }
 
+  /**
+   * Initiate multi algorithm coin selection.
+   */
+
+  async attemptSelection() {
+    // calculate cost of creating and spending a change output
+    // add dummy output for change.
+    const change = new Output();
+
+    if (this.changeAddress) {
+      change.script.fromAddress(this.changeAddress);
+    } else {
+      // In case we don't have a change address,
+      // we use a fake p2pkh output to gauge size.
+      change.script.fromPubkeyhash(Buffer.allocUnsafe(20));
```

Comment:
True, this could be replaced with a p2wpkh address like below. Bcoin wallets are segwit by default so this makes sense. It will be a very rare case that this logic is run, the wallet will always pass in a change address

```suggestion
      change.script.fromProgram(0, Buffer.allocUnsafe(20));
```